Column info in error message not correct

eslint 1.0, not global.

In Atom:
![image](https://cloud.githubusercontent.com/assets/713074/9100572/6d642762-3b91-11e5-9275-674ada102745.png)

In CLI:
src/index.js
  16:19  error  Infix operators must be spaced  space-infix-ops
  16:23  error  Extra semicolon                 semi

Seems column information is wrong.

--- a/lib/linter-eslint.coffee
+++ b/lib/linter-eslint.coffee
@@ -109,14 +109,12 @@ module.exports =
             allowUnsafeNewFunction ->
               results = linter
                 .verify TextEditor.getText(), config, filePath
-                .map ({message, line, severity, ruleId}) ->
+                .map ({message, line, severity, ruleId, column}) ->
 
-                  # Calculate range to make the error whole line
-                  # without the indentation at begining of line
                   indentLevel = TextEditor.indentationForBufferRow line - 1
-                  startCol = TextEditor.getTabLength() * indentLevel
-                  endCol = TextEditor.getBuffer().lineLengthForRow line - 1
-                  range = [[line - 1, startCol], [line - 1, endCol]]
+                  startCol = column || TextEditor.getTabLength() * indentLevel
+                  endOfLine = TextEditor.getBuffer().lineLengthForRow line - 1
+                  range = [[line - 1, startCol], [line - 1, endOfLine]]
 
                   if showRuleId
                     {

